activestate.windows.yaml

```diff
@@ -11,6 +11,8 @@ constants:
     value: state-update-dialog.exe
   - name: SYSTRAY_BUILDFLAGS
     value: -ldflags="-s -w -H=windowsgui"
+  - name: SVC_BUILDFLAGS
+    value: -ldflags="-s -w -H=windowsgui"
```

Comment:
Why was this necessary for this PR?

Reply:
This prevents a window from opening when the service autostarts on Windows.